src/Components/Admins/Form/index.jsx

```diff
@@ -242,9 +248,17 @@ const Form = () => {
           />
         </div>
         <div className={styles.modalBtns}>
-          <Link to="/admins/home/profile">
-            <Button text={'Cancel'} variant={'white'} />
-          </Link>
+          {location.pathname.includes('admins/home/form') ? (
```

Comment:
You can't add another condition inside a ternary like this. It should be:
```
          {location.pathname.includes('/superadmins/home/admins/add') ? (
            <Link to="/superadmins/home/admins">
              <Button text={'Cancel'} variant={'white'} />
            </Link>
          ) : (
            <Link to="/admins/home/profile">
              <Button text={'Cancel'} variant={'white'} />
            </Link>
          )}
```

Reply:
Fixed! Not exactly like this, but still!